content/plugins/pof-importer/pof-importer.php

```diff
@@ -68,27 +126,30 @@ public static function activate() {
         wp_schedule_event( time(), 'hourly', 'daily_cron' );
     }
     public static function deactivate() {
-        wp_clear_scheduled_hook('daily_cron');
+        wp_clear_scheduled_hook( 'daily_cron' );
     }
 
     // Setup cronjob for importer daily automatic execution
-    public function importer_cron () {
-        ini_set('memory_limit', '-1');
-        $this->tree_url = get_field('ohjelma-json', 'option');
-        $this->tips_url = get_field('tips-json', 'option');
+    public function importer_cron() {
+        $start = microtime( true );
+        $this->wp_cli_msg( 'Beginning import' );
+        ini_set( 'memory_limit', '-1' );
+        $this->tree_url = get_field( 'ohjelma-json', 'option' );
+        $this->tips_url = get_field( 'tips-json', 'option' );
         $this->import();
-        update_field( 'field_57c6b36acafd4', date("Y-m-d H:i:s"), 'option' );
+        update_field( 'field_57c6b36acafd4', date( 'Y-m-d H:i:s' ), 'option' );
```

Comment:
Consider moving this into a class constant in future branches. Then you could givei is good name describing the field. Example:
```
const SOME_IMPORTER_FIELD = 'field_57c6b36acafd4';
// And then..
update_field( static::SOME_IMPORTER_FIELD, date( 'Y-m-d H:i:s' ), 'option' );
```